internal/cli/atlas/backup/restores_start.go

```diff
@@ -55,6 +54,13 @@ func (opts *RestoresStartOpts) initStore() error {
 	return err
 }
 
+var automatedTemplate = "Restoring {{.TargetClusterName}} using snapshot {{.SnapshotID}}\n"
+var pointInTimeTemplate = "Restoring {{.TargetClusterName}} using point in time\n"
+var downloadTemplate = `Links to download the snapshot:
+REL	HREF{{range .Links}}
+{{.Rel}}	{{.Href}}{{end}}
```

Comment:
According to the docs the download links come in `deliveryUrl`, `Links` is usually a reference to the resource or resources

Reply:
yes, you are right. The problem is that I don't have that field in the response. The doc says 
```
NOTE

If empty, Atlas is processing the restore job. Use the Get All Cloud Backup Restore Jobs endpoint periodically check for a deliveryUrl download value for the restore job.
```
but my atlas is not restoring any job. 🤔

Reply:
Response:

```
{
  "id": "5f2175f5cad06326fabb828d",
  "snapshotId": "5f21627f8f4f772aff4f3da3",
  "deliveryType": "download",
  "expiresAt": "2020-07-29T17:13:25Z",
  "links": [
    {
      "rel": "self",
      "href": "https://cloud-dev.mongodb.com/api/atlas/v1.0/groups/5e4e593f70dfbf1010295836/clusters/Cluster0/backup/restoreJobs/5f2175f5cad06326fabb828d"
    },
    {
      "rel": "http://mms.mongodb.com/snapshot",
      "href": "https://cloud-dev.mongodb.com/api/atlas/v1.0/groups/5e4e593f70dfbf1010295836/clusters/Cluster0/backup/snapshots/5f21627f8f4f772aff4f3da3"
    },
    {
      "rel": "http://mms.mongodb.com/cluster",
      "href": "https://cloud-dev.mongodb.com/api/atlas/v1.0/groups/5e4e593f70dfbf1010295836/clusters/Cluster0"
    },
    {
      "rel": "http://mms.mongodb.com/group",
      "href": "https://cloud-dev.mongodb.com/api/atlas/v1.0/groups/5e4e593f70dfbf1010295836"
    }
  ],
  "timestamp": "2020-07-29T11:52:54Z"
}
```

Reply:
then If I run again the same mongocli command I get:

`Error: POST https://cloud-dev.mongodb.com/api/atlas/v1.0/groups/5e4e593f70dfbf1010295836/clusters/Cluster0/backup/restoreJobs: 400 (request "Bad Request") Cannot create a manual download of a snapshot when there is already an active download for that snapshot.`

Reply:
maybe here I can just change the template to `Manual download created for snapshot {{.SnapshotID}}`

Reply:
I think I know what is the problem here. As you can see from the picture below, atlas needs time to create the download link. So we are not going to have the download link in the response of this command.

I think we should update the template to `Manual download created for snapshot {{.SnapshotID}}`. 
Let me know what you think.

<img width="1288" alt="Screenshot 2020-07-29 at 15 25 33" src="https://user-images.githubusercontent.com/5663078/88805691-c1da1b80-d1af-11ea-8f5d-88276c371f46.png">

Reply:
I think given the circumstances I would simplify the template even more, we can just return the ID of the JOB and let them query it with list or describe